main.go

```diff
@@ -80,48 +56,18 @@ func main() {
 }
 
 func realMain() error {
-	cfg, err := parseConfig(os.Args[1:])
+	cfg, mut, err := parseFlags(os.Args[1:])
```

Comment:
Throughout, need a new name now that this is no longer 'Mutation'.

'mod', perhaps?